### ISSUE TITLE
POST request should have a header

### ISSUE THREAD

**Issue by s-n-o-v** [alexander-hagen/node-api-huobi#1]
Please note https://huobiapi.github.io/docs/spot/v1/en/#q8-why-the-api-return-39-gateway-internal-error-39

**Comment by alexander-hagen** [comment]
@s-n-o-v Please help to test. I have insufficient assets on Huobi to test POST requests. Fix is in v0.1.3

**Comment by s-n-o-v** [comment]
Hello, @alexander-hagen ! Thank you for fast response!

Yes, I've tested that code and I can do buy/sell actions. If you need to test something else please tell me and we will do that. I have some funds on Huobi

I saw your code updates, looks good. but please let me give an advice: you don't need to mix async/await and promises.
Also I have error response as example from Huobi, for that I've update back otherQuery function in privateApi.js (to get that response only). So, I got this...

```
  {
    status: 'error',
    'err-code': 'gateway-internal-error',
    'err-msg': 'Failed to load data. Try again later.',
    data: null
  }
```

And I updated your code using Async/await only. Please check that

```
    try {

        const res = await axios(options);

        return res.data;

    } catch (err) {

        if(err.hasOwnProperty("err-code")) {
            return {
                status: err.status,
                error: err['err-code'],
                reason: err['err-msg'],
                data: options
            };
        } else {
            return {
                status: err.status,
                data: options
            };
        };
    }
```
It works for me
